### PR TITLE
Update to bitflags 2.0

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

**Comment by sdroege** [comment]
Blocked by https://github.com/bitflags/bitflags/issues/308

**Comment by sdroege** [comment]
This could be revived now but I have other things to do right now :)

File: gdk-pixbuf/src/auto/pixbuf_animation.rs [hunk 1]
```diff
@@ -137,7 +137,12 @@ impl PixbufAnimation {
     }
 }
 
-pub trait PixbufAnimationExt: IsA<PixbufAnimation> + 'static {
+mod sealed {
```
Comment (sdroege) [inline gdk-pixbuf/src/auto/pixbuf_animation.rs:140]: This is blocked by someone updating all the manual traits accordingly (CC @jf2048 )

File: .gitmodules [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [submodule "gir"]
 	path = gir
-	url = https://github.com/gtk-rs/gir
+	url = https://github.com/sdroege/gir
```
Comment (GuillaumeGomez) [inline .gitmodules:3]: Don't forget to revert this one.

**Review by GuillaumeGomez — APPROVED** [review]
Looks good to me update the submodule update.

File: .gitmodules (hunk 2 above)
Comment (sdroege) [reply .gitmodules:3]: Yeah that's why those commits are marked WIP :)

**Comment by bilelmoussaoui** [comment]
Closing in favour of https://github.com/gtk-rs/gtk-rs-core/pull/1125

**Pull request closed without merging**